MultipleFileUpload/Model/MySQL/Queue.php

```diff
@@ -0,0 +1,162 @@
+<?php
```

Comment:
namespace should be names as the target for the bridge. You you name namespace mysql, I would expect to that this driver will use mysql_* functions